src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java

```diff
@@ -87,6 +91,13 @@ private DwpResponseDocument buildDwpResponseDocumentWithDate(String documentType
                 ).build());
     }
 
+    private void clearTransientFields(PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse) {
+        SscsCaseData sscsCaseData = preSubmitCallbackResponse.getData();
+        // Ensure that we always clear the dwpUploadResponseDynamicBenefitType as this is a temporary
```

Comment:
Think this comment is out of date now - needs to say 'dynamicBenefitType' instead of 'dwpUploadResponseDynamicBenefitType'

Reply:
Updated comment